src/app/models/saved_function.rb

```diff
@@ -0,0 +1,4 @@
+class SavedFunction < ApplicationRecord
```

Comment:
[Q] do you think it would make sense if we use the name `save` so the functions has many saves, and the user has many saves?

Reply:
Yes you're right. I'm changing it.